src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+public class Car {
+
+    private static final Integer MOVE_MINIMUM_CONDITION = 4;
+    private static final Integer MOVE_MAXIMUM_CONDITION = 9;
+
+    private Integer position = 0;
+
+    public void move(RandomValueGenerator randomValueGenerator) {
+        if (judgeForth(randomValueGenerator.generate())) {
+            position++;
+        }
+    }
+
+    public boolean judgeForth(int degree) {
```

Comment:
```suggestion
    private boolean judgeForth(int degree) {
```
public이 아닌 private으로 구현하는 것은 어떨까?